client/src/Components/Pages/Questionnaire/PersonalDetails1/index.js

```diff
@@ -4,45 +4,75 @@ import Select from 'react-select';
-    { value: 'technology', label: 'technology' },
+    { value: 'History', label: 'History' },
+    { value: 'Geography', label: 'Geography' },
+    { value: 'Design & technology', label: 'Design & technology' },
+    { value: 'Business Studies', label: 'Business Studies' },
+    { value: 'Chemistry', label: 'Chemistry' },
+    { value: 'Classics', label: 'Classics' },
+    { value: 'Computer Science', label: 'Computer Science' },
+    { value: 'Drama', label: 'Drama' },
+    { value: 'Economics', label: 'Economics' },
+    { value: 'Business Management', label: 'Business Management' },
+    { value: 'Health and Social Care', label: 'Health and Social Care' },
+    { value: 'ICT', label: 'ICT' },
+    { value: 'Latin', label: 'Latin' },
+    { value: 'Law', label: 'Law' },
+    { value: 'Media Studies', label: 'Media Studies' },
+    { value: 'Modern Foreign Languages', label: 'Modern Foreign Languages' },
+    { value: 'Music', label: 'Music' },
+    { value: 'Physical Education', label: 'Physical Education' },
+    { value: 'Politics', label: 'Politics' },
+    { value: 'Psychology', label: 'Psychology' },
+    { value: 'Religious Studies', label: 'Religious Studies' },
+    { value: 'Philosophy', label: 'Philosophy' },
+    { value: 'Social Sciences', label: 'Social Sciences' },
+    { value: 'Statistics', label: 'Statistics' },
+    { value: 'Welsh', label: 'Welsh' },
+    { value: 'Other', label: 'Other' },
   ];
 
 class PersonalDetails1 extends Component {
 
     state = {
-        selectedOption: null, 
+        selectedOption: [], 
     }
 
     handleChange = (selectedOption) => {
 
-        this.setState({ selectedOption });
+        this.setState();
+        
       }
       
     render() {
-        const { selectedOption } = this.state; 
         return ( 
-            <div>
+            <div className="container">
                 <div className="third-section">
-                    <h2>Now we’ve got the boring stuff out of the way, let’s get to know you better</h2>
+                    <h2>Now we've got the boring stuff out of the way, let's get to know more about your school studies</h2>
                     <div className="six-card">
                         <p>What subjects are you studying? Tick all that apply</p>
                         <Select
-                            value={selectedOption}
-                            isMulti={this.handleChange}
+                            onChange={this.handleChange}
+                            value={this.selectedOption}
+                            isMulti={true}
                             options={options}
                         />
                         <p>What are your favourite subjects Azara ? Tick up to 3</p>
-                        <input type="text" className ="sec1-input"/>
+                        <Select
+                            onChange={this.handleChange}
+                            value={this.selectedOption}
```

Comment:
Here you'll need `this.state.(your state attribute name)` again, instead of this.selectedOption.